Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -1,26 +1,43 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Collections.Generic;
+
+namespace BootCamp.Chapter
 {
     public class Inventory
     {
-        private Item[] _items;
-        public Item[] GetItems()
-        {
-            return new Item[0];
-        }
+        private const int defaultInventorySize = 10;
+
+        public List<Item> Items { get; private set; }
```

Comment:
If you set something in ctor, it doesn't need a private setter. Can be just a readonly (`get;`) property.

Reply:
Thank you! I'll keep this in mind!